app/components/chart-footnotes.tsx

```diff
@@ -211,7 +211,10 @@ export const ChartFootnotes = ({
                     <DataDownloadMenu
                       dataSource={dataSource}
                       title={dataCubeMetadata.title}
-                      filters={filters}
+                      filters={
+                        filters ??
+                        chartConfig.cubes.map((cube) => ({ iri: cube.iri }))
+                      }
```

Comment:
Not sure I understand this ? Shouldn't we also "mix" the filters with the cubes ?

Reply:
Here we do not use getFullDataDownloadFilters  ?

Reply:
Hmmm, here we either pass `filters` if they are present, or "construct" them in case they are missing. In `getFullDataDownloadFilters` we need to pass `filters` as an argument, so couldn't apply this to `chartConfig.cubes.map((cube) => ({ iri: cube.iri }))`. Let me know if it makes sense or if I missed something 👀

Reply:
I refactored a bit to introduce a new function, let me know if it makes more sense now @ptbrowne 🙇‍♂️